model-datastructure/src/commonTest/kotlin/LinearHistoryTest.kt

```diff
@@ -57,133 +57,6 @@ class LinearHistoryTest {
         assertEquals(expected, actual)
     }
 
-    @Test
-    fun knownPerformanceIssue() {
-        // This test was dumped from actual case discovered during a profiling session.
-
-        val v30000003a = version(12884901946, null)
```

Comment:
> The `knownPerformanceIssue` test case was now constantly timing out when
being executed in JS. After clarification, all corner cases it included
are already covered by the **remaining tests in the test file**. Therefore,
I have dropped this test case to avoid the timeout and to reduce the
amount of hard to maintain test cases.

Actually not all tests are in this file.
Other relevant test include:
* MergeOrderTest
* ConflictResolutionTest
* UndoTest

See https://github.com/modelix/modelix.core/pull/354

Reply:
I've reworded the commit message to include the tests you mentioned.